### DISCUSSION TITLE
[@types/architect__functions] table types are out of date

### DISCUSSION THREAD

**Discussion by lpsinger** [DefinitelyTyped/DefinitelyTyped#63652] [Issues with a @types package] [answered]
As of @architect/functions 5.3.0, ArcTables.doc and ArcTables.db should return the AWS v3 SDK and not the AWS v2 SDK if the project is targeting the AWS nodejs18.x runtime.

I don't know how to make the types depend on the AWS runtime. Perhaps it would be good enough if the types depended upon the targeted version of Node.js?

[a comment by an automated account is not shown]

**Comment by lpsinger** [comment, marked as answer]
Never mind. These types are now included inside @architect/functions itself. See architect/functions#540.

**Comment by lpsinger** [comment]
How about deprecating `@types/architect__functions` to avoid confusion?

**Comment by activescott** [comment]
I'm down to deprecate. If someone needs my approval just tag me
